src/sdk/translations/ifw_zh_CN.ts

```diff
@@ -1286,65 +1285,65 @@ Error while loading %2</source>
     </message>
     <message>
         <source>Failed to write settings.</source>
-        <translation>写入设置失败。</translation>
+        <translation>未能写入设置。</translation>
     </message>
 </context>
 <context>
     <name>QInstaller::InstallIconsOperation</name>
     <message>
         <source>&lt;source path&gt; [vendor prefix]</source>
-        <translation>&lt;源路径&gt; [vendor prefix]</translation>
+        <translation>&lt;source path&gt; [vendor prefix]</translation>
```

Comment:
Translation missing

Reply:
These translations were removed on purpose by the translators